test/sources/compile_deps.t/run.t

```diff
@@ -11,18 +11,18 @@ compile-deps
 [a.cmti] does not depend on B, while its implementation [a.cmt] depends on B.
 
   $ odoc compile-deps a.cmti
-  CamlinternalFormatBasics 8f8f634558798ee408df3c50a5539b15
-  Stdlib 79b0e9d3b6f7fed07eb3cc2abb961b91
   A 21e6137bd9b3aaa3c66960387b5f32c0
-  $ odoc compile-deps a.cmt
-  Stdlib 79b0e9d3b6f7fed07eb3cc2abb961b91
   CamlinternalFormatBasics 8f8f634558798ee408df3c50a5539b15
-  B 903ddd9b7c0fa4ee6d34b4af6358d1e1
+  Stdlib 6d7bf11af14ea68354925f3a37387930
+  $ odoc compile-deps a.cmt
   A 21e6137bd9b3aaa3c66960387b5f32c0
+  B 903ddd9b7c0fa4ee6d34b4af6358d1e1
+  CamlinternalFormatBasics 8f8f634558798ee408df3c50a5539b15
```

Comment:
This looks very brittle! I guess it's OK until we merge the stable anchors though.